packages/web-shared/components/Searchbar/Autocomplete.js

```diff
@@ -395,21 +395,37 @@ export default function Autocomplete({
       closeDropdownMenu();
     }
 
-    setAriaSelectedToFalseOnHover('.aa-List', '.aa-Item', '.empty-feed');
+    setAriaSelectedToFalseOnHover(".aa-List", ".aa-Item", ".empty-feed");
 
-    document.addEventListener('click', handleClickOutside);
+    document.addEventListener("click", handleClickOutside);
 
     return () => {
-      document.removeEventListener('click', handleClickOutside);
+      document.removeEventListener("click", handleClickOutside);
     };
   }, [autocompleteState.isOpen, autocomplete, setShowTextPrompt]);
 
+  const searchResults = [];
```

Comment:
Instead of looping through and remapping the results I think you can just change the order of `getSources()`
```suggestion
       getSources() {
        return [
          // (3) Use an Algolia index source.
          {
            sourceId: 'pages',
            getItemInputValue({ item }) {
              return item.query;
            },
            getItems({ query }) {
              return getAlgoliaResults({
                searchClient,
                queries: [
                  {
                    indexName: `WebPages_Global`,
                    query,
                    params: {
                      facetFilters: [`church:${searchState.church}`],
                      hitsPerPage: 4,
                      clickAnalytics: true,
                      // highlightPreTag: '<mark>',
                      // highlightPostTag: '</mark>',
                    },
                  },
                ],
              });
            },
            getItemUrl({ item }) {
              return `/?query=${item.name}`;
            },
          },
          {
            sourceId: 'content',
            getItemInputValue({ item }) {
              return item.query;
            },
            getItems({ query }) {
              return getAlgoliaResults({
                searchClient,
                queries: [
                  {
                    indexName: `ContentItem_${searchState.church}`,
                    query,
                    params: {
                      hitsPerPage: 4,
                      clickAnalytics: true,
                      // highlightPreTag: '<mark>',
                      // highlightPostTag: '</mark>',
                    },
                  },
                ],
              });
            },
            getItemUrl({ item }) {
              return `/?query=${item.name}`;
            },
          },
        ];
      },
```